buildSrc/src/main/groovy/com/uber/okbuck/core/model/Scope.groovy

```diff
@@ -87,9 +94,10 @@ class Scope {
                 configuration.files.findAll { File resolved ->
                     !resolvedFiles.contains(resolved)
                 }.each { File localDep ->
+                    String localDepPath = rootPath.relativize(Paths.get(localDep.absolutePath)).toString()
```

Comment:
You can just use Fileutil.relativePath from okbuck here instead of bringing in all new imports. we can refactor that util method later to use the nio apis.

using nio makes this plugin only work if you have jdk 1.8